debug.py

```
@@ -20,12 +22,15 @@ async def main() -> None:  # noqa: D103
         connected = await elegoo_printer.connect_printer()
         if connected:
             logger.debug("Polling Started")
-            elegoo_printer.set_printer_video_stream(toggle=True)
+            await asyncio.sleep(2)
+            elegoo_printer.set_printer_video_stream(toggle=False)
+            # logger.debug(elegoo_printer.get_current_print_thumbnail())
```

Ruff (0.8.2)

Found commented-out code

Remove commented-out code

(ERA001)